h2o-core/src/test/java/water/runner/H2ORunner.java

```diff
@@ -129,6 +129,11 @@ private void printLeakedKeys(final Key[] leakedKeys) {
                         Log.err(String.format("       Chunk id %d, key '%s'", i, chunkKey));
                         leakedKeysSet.remove(chunkKey);
                     }
+                    
+                    if (leakedKeysSet.contains(vec.rollupStatsKey())) {
```

Comment:
Also print and group leaked rollup stats, otherwise those show up in the "other leaked keys" section.